tests/src/Interop/WinRT/NativeComponent/CMakeLists.txt

```diff
@@ -0,0 +1,114 @@
+cmake_minimum_required (VERSION 2.8.12)
+project (WinRTNativeComponent)
+include_directories( ${INC_PLATFORM_DIR} )
+include ("${CLR_INTEROP_TEST_ROOT}/Interop.cmake") 
+include_directories("../Contracts")
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+set(SOURCES
+  Component.Contracts.ArrayTesting.cpp
+  Component.Contracts.BindingProjectionsTesting.cpp
+  Component.Contracts.BindingViewModel.cpp
+  Component.Contracts.BooleanTesting.cpp
+  Component.Contracts.ExceptionTesting.cpp
+  Component.Contracts.KeyValuePairTesting.cpp
+  Component.Contracts.NullableTesting.cpp
+  Component.Contracts.StringTesting.cpp
+  Component.Contracts.TypeTesting.cpp
+  Component.Contracts.UriTesting.cpp
+)
+
+set (MIDL_GENERATED_WINMD ${CMAKE_CURRENT_BINARY_DIR}/WinMD/Component.Contracts.winmd)
+file(TO_NATIVE_PATH ${MIDL_GENERATED_WINMD} MIDL_GENERATED_WINMD_WINDOWS_PATH)
+
+if(NOT DEFINED CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION OR CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION STREQUAL "" )
+  message(FATAL_ERROR "Windows SDK is required for the WinRT interop tests build.")
+else()
+  message("Using Windows SDK version ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION} for WinRT interop tests.")
+endif()
+
+set (WINDOWS_KITS_10_ROOT "C:/Program Files (x86)/Windows Kits/10/")
```

Comment:
This is very very bad juju. Windows Kits must be installed on the system drive, but the system drive can be any drive letter. Instead consume the appropriate environment variable (e.g. `%ProgramFiles(x86)%`). Alternatively, I believe that the developer command prompt is used during CMake project gen, if so use `%WindowsSdkDir%`.